java/code/src/com/suse/manager/reactor/utils/test/dummy_packages_redhatprodinfo_centos_sll.json

```diff
@@ -0,0 +1,101 @@
+{
+  "cmd_|-respkgquery_|-rpm -q --whatprovides 'sles_es-release-server'_|-run": {
+    "comment": "onlyif execution failed",
+    "__id__": "respkgquery",
+    "name": "rpm -q --whatprovides 'sles_es-release-server'",
+    "duration": 86.271,
+    "__run_num__": 3,
+    "start_time": "10:30:07.714911",
+    "skip_watch": true,
+    "changes": {},
+    "result": true
+  },
+  "cmd_|-respkgquery_|-rpm -q --whatprovides 'sll-release'_|-run": {
+    "comment": "onlyif execution failed",
+    "__id__": "respkgquery",
+    "name": "rpm -q --whatprovides 'sll-release'",
+    "duration": 86.271,
+    "__run_num__": 3,
+    "start_time": "10:30:07.714911",
+    "skip_watch": true,
+    "changes": {},
+    "result": true
+  },
+  "cmd_|-centosrelease_|-cat /etc/centos-release_|-run": {
+    "comment": "Command \"cat /etc/centos-release\" run",
+    "name": "cat /etc/centos-release",
+    "start_time": "10:30:07.679765",
+    "result": true,
+    "duration": 22.903,
+    "__run_num__": 1,
+    "changes": {
+      "pid": 15301,
+      "retcode": 0,
+      "stderr": "",
+      "stdout": "CentOS Stream release 9.0 (Name)"
```

Comment:
It's only a test, but we do not support CentOS 9 (Stream). SLL needs to come together with something else than CentOS. Maybe Rocky or Alma. I am not 100% sure what is a supported variant. Maybe "standalone" SLL is now also an option.

Reply:
@mcalmer is right.

I just checked with Liberty team and they mentioned that as a part of Liberty they support RHEL/CentOS(except CentOS stream), and Oracle Linux upon request.